GenumKit/Stencil/Filters.swift

```diff
@@ -80,12 +80,32 @@ struct StringFilters {
 
 struct ArrayFilters {
   static func join(value: Any?) throws -> Any? {
-    guard let array = value as? [Any] else { throw FilterError.InvalidInputType }
-    let strings = array.flatMap { $0 as? String }
-    guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    let strings: [String]
+    if let array = value as? [Any] {
+      strings = array.flatMap { $0 as? String }
+      guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    } else if let array = value as? [String] {
+      strings = array
+    } else {
+      throw FilterError.InvalidInputType
+    }
 
     return strings.joinWithSeparator(", ")
   }
+
+  static func joinAnd(value: Any?) throws -> Any? {
```

Comment:
Is the problem that Template isn't actually `open`? Because I'm confused. For me on 0.7.1:

`public class Template: ExpressibleByStringLiteral`
https://github.com/kylef/Stencil/blob/master/Sources/Template.swift